### PR TITLE
Open API 3 Path Parameter & Delete Standard Rule Check Failed Issue Fixed  

### PR DESCRIPTION
**Pull request by hafijul233**
Changes this PR has these changes for [BUG] #266 
1. **When exporting an OpenAPI3 JSON the Path Parameter values are also generated as query values**
2. **Excluding request body while generating the delete request. As OpenAPI3 discourages request body**
3. **Description Content added from configuration file**
4. **Exclude certain HTTP methods from exporting into JSON. Ex: `HEAD`**

### CODE REVIEW

File: src/LaravelRequestDocsToOpenApi.php [hunk 1]
```diff
@@ -33,9 +33,18 @@ public function openApi(array $docs): LaravelRequestDocsToOpenApi
     private function docsToOpenApi(array $docs): void
     {
         $this->openApi['paths'] = [];
+        $includeBodyForDelete   = config('request-docs.open_api.include_body_on_delete', false);
+        $excludeHttpMethods     = array_map(fn ($item) => strtolower($item), config('request-docs.open_api.exclude_http_methods', []));
+
         foreach ($docs as $doc) {
-            $requestHasFile  = false;
```
Comment (kevincobain2000) [inline src/LaravelRequestDocsToOpenApi.php:37]: This declaration is necessary 

https://github.com/rakutentech/laravel-request-docs/blob/64325d680b1d2bd738300f9e61073b869ca6b4b3/src/LaravelRequestDocsToOpenApi.php#L64-L75

**Review by kevincobain2000 — APPROVED** [review]
Can you also check this please @kitloong

File: src/LaravelRequestDocsToOpenApi.php (hunk 1 above)
Comment (kevincobain2000) [reply src/LaravelRequestDocsToOpenApi.php:37]: Sorry it has been moved down.

File: config/request-docs.php [hunk 2]
```diff
@@ -58,13 +58,17 @@
         'description' => 'Laravel Request Docs',
         // default version that this library provides
         'version' => '3.0.0',
-        // changeable
+        //changeable
```
Comment (kitloong) [inline config/request-docs.php:61]:
```suggestion
        // changeable
```

File: config/request-docs.php [hunk 3]
```diff
@@ -58,13 +58,17 @@
...
         'document_version' => '1.0.0',
         // license that you want to display
         'license' => 'Apache 2.0',
         'license_url' => 'https://www.apache.org/licenses/LICENSE-2.0.html',
         'server_url' => env('APP_URL', 'http://localhost'),
-
+        //openapi 3.0.x doesn't support request body for develop operation
```
Comment (kitloong) [inline config/request-docs.php:67]:
```suggestion
        // openapi 3.0.x doesn't support request body for develop operation
```

Comment (kitloong) [reply config/request-docs.php:67]:
```
... for develop operation
```
Do you mean `delete` ?

File: config/request-docs.php [hunk 4]
```diff
@@ -58,13 +58,17 @@
...
+        //ref: https://github.com/OAI/OpenAPI-Specification/pull/2117
```
Comment (kitloong) [inline config/request-docs.php:68]:
```suggestion
        // ref: https://github.com/OAI/OpenAPI-Specification/pull/2117
```

File: config/request-docs.php [hunk 5]
```diff
@@ -58,13 +58,17 @@
...
+        'include_body_on_delete' => false,
```
Comment (kitloong) [inline config/request-docs.php:69]:
```suggestion
        'delete_with_body' => false,
```

File: config/request-docs.php (hunk 3 above)
Comment (hafijul233) [reply config/request-docs.php:67]: sorry, that's a typo. yes for delete operation

File: config/request-docs.php (hunk 5 above)
Comment (hafijul233) [reply config/request-docs.php:69]: Okay. I will commit this change

Comment (hafijul233) [reply config/request-docs.php]: I have push those corrections.[Commit Details](https://github.com/hafijul233/laravel-request-docs/commit/669a0333d8c8edde47260dc3e53c9fbe962ebc09)

**Review by kitloong — APPROVED** [review]
Thank you, LGTM

**Pull request merged**